### PR TITLE
feat: initilize npm project

### PR DESCRIPTION
**Pull request by abyssparanoia**
## Proposal

- initilize npm project
- using typescript as developing language
- using eslint as linter
- using prettier as formatter
- using jest as testing framework
- add issue and pull request templates

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,70 @@
+{
+	"name": "devdev-kit-js",
```
Comment (abyssparanoia) [inline package.json]: please check npm name.

We need to decide the name to actually publish.

File: package.json [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+		"husky": "^3.0.8",
+		"jest": "^24.9.0",
+		"lint-staged": "^9.4.2",
+		"pre-commit": "^1.2.2",
+		"prettier": "^1.18.2",
+		"rimraf": "^3.0.0",
+		"ts-jest": "^24.1.0",
+		"typescript": "^3.6.3"
+	},
+	"husky": {
```
Comment (abyssparanoia) [inline package.json]: Use husky to force format and lint

File: package.json (hunk 1 above)
Comment (aggre) [reply package.json]: Yeah, I think the name is `@devprotocol/dev-kit-js`.

File: package.json (hunk 2 above)
Comment (aggre) [reply package.json]: Nice!

File: package.json (hunk 1 above)
Comment (abyssparanoia) [reply package.json]: I fixed it

**Comment by aggre** [comment]
Nice configuration!

**Pull request merged**